docs/triggers/azure-event-hubs.md

```diff
@@ -0,0 +1,88 @@
+# Azure Event Hubs
+
+Azure Event Hubs Trigger allows a sensor to publish events to [Azure Event Hubs](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-about). Argo Events integrates with Azure Event Hubs to stream data from an `EventSource`
+
+**NOTE:** Parametrization for `fqdn` and `hubName` values are not yet supported. 
+
+## Specification
+The Azure Event Hubs trigger specification is available [here](https://github.com/argoproj/argo-events/blob/master/api/sensor.md#azureeventhubstrigger).
+
+## Send an Event to Azure Event Hubs
+1. Make sure to have the eventbus deployed in the namespace.
+
+1. [Create an event hub](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create)
+
+1. Make sure that the Shared Access Key used to connect to Azure Event Hubs has the `Send` policy.
+
+1. Get the `Primary Key` of the Shared Access Policy, the `Name` of the Shared Access Policy, the `Hub Name`, and the `FQDN` of the Azure Event Hubs Namespace
+
+1. Create a secret called `azure-event-hubs-secret` as follows:
+    
+    **NOTE: `sharedAccessKey` refers to the `Primary Key` and `sharedAccessKeyName` refers to the Name of the Shared Access Policy.**
+
+        apiVersion: v1
+        kind: Secret
+        metadata:
+          name: azure-event-hubs-secret
+        type: Opaque
+        data:
+          sharedAccessKey: <base64-shared-access-key>
+          sharedAccessKeyName: <base64-shared-access-key-name>
+
+1. Let's set up a webhook event-source to process incoming requests.
+
+        kubectl apply -n argo-events -f https://raw.githubusercontent.com/argoproj/argo-events/stable/examples/event-sources/webhook.yaml
+
+1. Create the sensor with the following template. Replace the necessary values for `fqdn` and `hubName`:
+
+        apiVersion: argoproj.io/v1alpha1
+        kind: Sensor
+        metadata:
+        name: azure-events-hub
+        spec:
+        template:
```

Comment:
template/serviceAccountName here is not needed here, a `default` service account would work.

Reply:
Removed service account reference